typhon/cloudmask/aster.py

```diff
@@ -471,6 +471,37 @@ def retrieve_cloudmask(
 
         return clmask
 
+    def read_coordinates(self, channel="1"):
+        """Read reduced latitude and longitude grid.
+
+        Extract geolocation table containing latitude and longitude values at
+        11 x 11 lattice points. Latitudes are provided in geocentric coordinates
+        and are maped to geodetic values.
+
+        Parameters:
+            channel (str): ASTER channel number. '1', '2', '3N', '3B', '4',
+                    '5', '6', '7', '8', '9', '10', '11', '12', '13', '14'.
+
+        Returns:
+            tuple(ndarray, ndarray): latitude, longitude
+        """
+
+        sdsidx = {
+            "VNIR": (0, 1),
+            "SWIR": (6, 7),
+            "TIR": (14, 15)
+        }
+
+        latstr = (f"HDF4_SDS:UNKNOWN:{self.filename}:{sdsidx[self.channel2sensor[channel]][0]}")
```

Comment:
Is there a reason for the parantheses around the f-string?

Reply:
No, but the string is too long :) I'll fix it